org.freedesktop.dabrain34.squared.json

```diff
@@ -0,0 +1,43 @@
+{
+  "app-id": "org.freedesktop.dabrain34.squared",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "squared.sh",
+  "finish-args": [
+    "--socket=pulseaudio",
+    "--device=dri",
+    "--share=ipc",
+    "--socket=x11",
+    "--persist=.local/share/godot/app_userdata/squared"
+  ],
+  "modules": [
+    {
+      "name": "squared",
+      "buildsystem": "meson",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://gitlab.freedesktop.org/dabrain34/squared/uploads/137574a448d09099dd229da691be3d83/squared-0.1.0.tar.xz",
+          "sha256": "78516beeaf56b5566fe57c827577ec9cd93e0b5e5eb29f6a20f82c0fb7b0f6e8"
+        },
+        {
+          "type": "file",
+          "url": "https://gitlab.freedesktop.org/dabrain34/squared/uploads/f52198b93be7f495a85c9f8e70c34c5e/squared_x86_64",
+          "sha256": "fd672ed5053a1a52998fa9a256347a2d91684a29fc061e4ba2d65c10919aa507"
+        },
+        {
+          "type": "file",
+          "url": "https://gitlab.freedesktop.org/dabrain34/squared/uploads/5fe7bf6de6f1d7a3c782822fd6b63732/data.pck",
+          "sha256": "5c47b5b0555de0a511886ff2272c5b4fc3b8a4c3422df79005de9d3b4d448cb5"
+        }
+      ],
+      "post-install": [
+        "install -Dm755 ../squared_x86_64 $FLATPAK_DEST/bin/squared",
+        "install -Dm555 ../data.pck $FLATPAK_DEST/bin/data.pck",
+        "echo (cd /app/bin && /app/bin/squared) > $FLATPAK_DEST/bin/squared.sh",
```

Comment:
this is invalid shell. 

There is the `script` source type for what you are trying to do.